doc/conf.py

```diff
@@ -139,8 +139,28 @@
         }
     ],
     "use_edit_page_button": True,
+    "show_powered_by": True,
     "gitter_room": "spyder-ide/public",
-    "alert_message": "This documentation is still a work in progress. Check out the <a href='https://docs.spyder-ide.org' target='_blank'>Spyder 3 docs</a> for the stable version"
+    "alert_message": "This documentation is still a work in progress. Check out the <a href='https://docs.spyder-ide.org' target='_blank'>Spyder 3 docs</a> for the stable version",
+    "open_collective": "spyder",
+    "footer_links": [
+        {
+            "url": "https://github.com/pandas-dev/pandas",
+            "name": "GitHub"
+        },
+        {
+            "url": "https://twitter.com/pandas_dev",
+            "name": "Twitter"
+        },
+        {
+            "url": "https://www.facebook.com/SpyderIDE/",
+            "name": "Facebook"
+        },
+        {
```

Comment:
```suggestion
        {
            "url": "https://instagram.com/spyderide",
            "name": "Instagram",
        },
        {
```

Missing our instagram that the main site has